packages/applicant/src/components/description-list/DescriptionList.tsx

```diff
@@ -16,13 +20,19 @@ export const DescriptionList: FC<DescriptionListProps> = ({
   data,
   needAddOrChangeButtons,
   needBorder,
+  organisationType,
 }) => {
   const router = useRouter();
   let pathname: string;
   if (router) {
     pathname = router.pathname;
   }
 
+  const removeOrganisationRowForIndividuals = (row: DescriptionListDataProps) =>
+    row.term !== ORGANISATION || organisationType !== INDIVIDUAL;
```

Comment:
```suggestion
    !(row.term === ORGANISATION && organisationType === INDIVIDUAL);
```
I think displaying the condition this way is a bit easier to read

Reply:
should be 

```js
!(row.term === ORGANISATION || organisationType === INDIVIDUAL);
```

right?

Reply:
Nah I think by applying De Morgans, `!A or !B` becomes `!(A and B)` - right? This condition is filtering out the Organisation row if it's an individual, so we are checking if the row matches both of those conditions and returning false if it does

Reply:
I am bad at boolean logic, deffo gonna take De Morgan's word for it